From dc2429062dd7e527b96cd03e5d2ca310df64e5c5 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Sat, 9 Feb 2019 18:35:00 +0300
Subject: [PATCH 21/34] drm/panel: s6e63m0: disable rotation

Low part of GTCON register is responsible for flipping/rotation.
This commit disables default rotation by 180 degrees.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 drivers/gpu/drm/panel/panel-samsung-s6e63m0.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c b/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
index 4312aa3e4386..dbd57239359e 100644
--- a/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
+++ b/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
@@ -200,7 +200,7 @@ static void s6e63m0_init(struct s6e63m0 *ctx)
 	s6e63m0_dcs_write_seq_static(ctx, MCS_DISPLAY_CONDITION,
 				     0x02, 0x03, 0x1c, 0x10, 0x10);
 	s6e63m0_dcs_write_seq_static(ctx, 0xf7,
-				     0x03, 0x00, 0x00);
+				     0x00, 0x00, 0x00);
 
 	s6e63m0_dcs_write_seq_static(ctx, MCS_GAMMA_CTRL,
 				     0x00, 0x18, 0x08, 0x24, 0x64, 0x56, 0x33,
-- 
2.20.1

